website/content/docs/connect/manage-traffic/route-to-local-upstreams.mdx

```diff
@@ -10,9 +10,9 @@ This topic describes how to enable locality-aware routing so that Consul can pri
 
 <EnterpriseAlert> This feature is available in Consul Enterprise. </EnterpriseAlert>
 
-!> **Warning**: Locality-aware routing is an advanced feature that may adversely impact service capacity if used incorrectly. By default, this feature routes 100% of traffic to the most local set of service instances, and failover only occurs when there are no healthy instances available in the most local set. You should only follow these instructions when every service within a zone has enough capacity to handle requests from downstream services within the same zone.
+!> **Warning**: Locality-aware routing is an advanced feature that may adversely impact service capacity if used incorrectly. By default, this feature routes 100% of traffic to the most local set of service instances, and failover only occurs when there are no healthy instances available in the most local set. You should only follow these instructions when each set of upstream instances (within the same zone, within the same region, and non-local) have enough capacity to handle requests from downstream service instances in their respective zone(s).
 
--> **Note**: It is possible to adjust the load balancing and failover behavior for this feature globally or per-service via the [Property Override Envoy extension](/consul/docs/connect/proxies/envoy-extensions/usage/property-override). Please familiarize with Envoy docs on [`overprovisioning_factor`](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/endpoint/v3/endpoint.proto#config-endpoint-v3-clusterloadassignment) and [outlier detection](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/outlier_detection.proto#config-cluster-v3-outlierdetection) before attempting to modify configuration.
+-> **Note**: It is possible to adjust the load balancing and failover behavior for this feature globally or per-service via the [Property Override Envoy extension](/consul/docs/connect/proxies/envoy-extensions/usage/property-override) by modifying [`overprovisioning_factor`](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/endpoint/v3/endpoint.proto#config-endpoint-v3-clusterloadassignment) and [outlier detection](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/outlier_detection.proto#config-cluster-v3-outlierdetection) configuration. By default, Consul sets `overprovisioning_factor` to `100000` (enforcing "total" failover) and `max_ejection_percent` to `100`. Please familiarize with Envoy documentation on these fields before attempting to modify them.
```

Comment:
```suggestion
```
This is in the wrong section

Reply:
Ok - where do you think it would best fit?